libs/features/georoutingv2/georoutingv2.css

```diff
@@ -1,15 +1,21 @@
 .dialog-modal.locale-modal-v2 .georouting-wrapper {
+  display: block !important;
```

Comment:
Is this absolutely necessary?

Reply:
Yes, I'm setting `display: none;` within javascript so this is necessary to override the inline style.